google-cloud-trace/docs/toc.json

```diff
@@ -1,8 +1,71 @@
 {
-  "guides": [],
+  "guides": [
+    {
+      "title": "Troubleshooting",
+      "id": "troubleshooting",
+      "edit": "https://github.com/GoogleCloudPlatform/gcloud-common/edit/master/troubleshooting/readme.md",
+      "contents": "https://raw.githubusercontent.com/GoogleCloudPlatform/gcloud-common/master/troubleshooting/readme.md"
+    },
+    {
+      "title": "Contributing",
+      "id": "contributing",
+      "edit": "https://github.com/GoogleCloudPlatform/gcloud-common/edit/master/contributing/readme.md",
+      "contents": "https://raw.githubusercontent.com/GoogleCloudPlatform/gcloud-common/master/contributing/readme.md"
+    }
+  ],
   "services": [
     {
-      "title": "TraceServiceApi",
+      "title": "Google::Cloud",
+      "type": "google/cloud"
+    },
+    {
+      "title": "Google::Cloud::Trace",
+      "type": "google/cloud/trace",
+      "nav": [
+        {
+          "title": "LabelKey",
+          "type": "google/cloud/trace/labelkey"
+        },
+        {
+          "title": "Middleware",
+          "type": "google/cloud/trace/middleware"
+        },
+        {
+          "title": "Notifications",
+          "type": "google/cloud/trace/notifications"
+        },
+        {
+          "title": "Project",
+          "type": "google/cloud/trace/project"
+        },
+        {
+          "title": "Railtie",
+          "type": "google/cloud/trace/railtie"
+        },
+        {
+          "title": "ResultSet",
+          "type": "google/cloud/trace/resultset"
+        },
+        {
+          "title": "Span",
+          "type": "google/cloud/trace/span"
+        },
+        {
+          "title": "SpanKind",
+          "type": "google/cloud/trace/spankind"
+        },
+        {
+          "title": "TimeSampler",
+          "type": "google/cloud/trace/timesampler"
+        },
+        {
+          "title": "TraceRecord",
+          "type": "google/cloud/trace/tracerecord"
+        },
+      ]
+    },
```

Comment:
Do you also want to add Trace as a dependency in the `google-cloud` umbrella package? (The other Stackdriver packages such as Monitoring and Error Reporting are already included.) In that case, copy this to be a nav item in `google-cloud/docs/toc.json`.